lib/bolt/util.rb

```diff
@@ -85,6 +85,38 @@ def first_run?
         Bolt::Config.user_path && !File.exist?(first_runs_free)
       end
 
+      def find_file_in_mod(mod, module_file)
+        subdir = module_file.split(File::SEPARATOR).first
+        case subdir
+        when 'scripts'
+          script_path = module_file.split(File::SEPARATOR).drop(1).join(File::SEPARATOR)
```

Comment:
`script_path` will be `""` if the original path was just `mymodule/scripts` (or `nil` if you use the `split(..., 2)` suggestion above) . In that case, we probably shouldn't fall back to `mod.script`. Ditto for the `files` case.